resources/content/config-content/cluster/content/etc/init.d/cluster

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+### BEGIN INIT INFO
+# Provides:		cluster
+# Default-Start:	2 3 4 5
+# Default-Stop:		0 1 6
+# Short-Description:	swarm cluster
+# Description:		swarm cluster
+### END INIT INFO
+
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+SWARM_ARG_FILE=/etc/swarm_args.conf
```

Comment:
/etc/cluster would be the better place. As most of the existing config items - monit, haproxy, ipsec - have dedicated folders under /etc dir